### PR TITLE
Add GPU pods count to project overview

### PR DESCRIPTION
**Pull request by rawagner**

### CODE REVIEW

File: src/components/ProjectOverview/GPUProjectInventory.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  const [pods, loaded, loadError] = useK8sWatchResource<Pod[]>({
+    groupVersionKind: {
+      kind: 'Pod',
+      version: 'v1',
+    },
+    isList: true,
+    namespace: projectName,
+  });
+
+  let title: React.ReactNode = t('{{count}} GPU pod', { count: pods.length });
```
Comment (mareklibra) [inline src/components/ProjectOverview/GPUProjectInventory.tsx:26]: Isn't that missing some kind of filtering?

Comment (rawagner) [reply src/components/ProjectOverview/GPUProjectInventory.tsx]: whops, fixed :)

**Comment by fabiendupont** [comment]
Is anything missing on this PR?

File: src/components/ProjectOverview/GPUProjectInventory.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+type GPUProjectInventoryProps = {
+  projectName: string;
+};
+
+const GPUProjectInventory = ({ projectName }: GPUProjectInventoryProps) => {
+  const { t } = useTranslation();
+  const [pods, loaded, loadError] = useK8sWatchResource<Pod[]>({
+    groupVersionKind: {
+      kind: 'Pod',
```
Comment (mareklibra) [inline src/components/ProjectOverview/GPUProjectInventory.tsx:19]: nit: we should put that into constants or reuse from model. It's repeating.

**Review by mareklibra — APPROVED** [review]

**Comment by rawagner** [comment]
@fabiendupont nope, thanks for the ping.

**Pull request merged**